spec/spec_helper.rb

```diff
@@ -26,4 +26,6 @@
   if config.files_to_run.one?
     config.default_formatter = "doc"
   end
+
+  config.order = :random
```

Comment:
Do we need this? It's already defined at [line 23](https://github.com/Shopify/tapioca/pull/13/files#diff-93830fa29d616f7c87903d08b5b1b29aR23).

Reply:
Removed.